scripts/create_test_subset.py

```diff
@@ -627,6 +658,30 @@ def get_sids_for_families(
     return included_sids
 
 
+def get_sids_for_cohorts(
+    project: str, cohorts: set[str], cohort_samples_n: int
+) -> set[str]:
+    """Returns cohort_samples_n specific samples for given cohort IDs."""
+
+    cohort_sid_output = query(COHORT_QUERY, {'project': project})
+
+    all_cohort_groups = cohort_sid_output.get('project', {}).get('cohorts', [])
+
+    all_cohorts_sample_ids_subset: set[str] = set()
+    for cohort in all_cohort_groups:
+        sids_for_cohort: list[str] = []
+        if cohort.get('id') in cohorts:
+            seq_groups = cohort.get('sequencingGroups', [])
+            for seq_group in seq_groups:
+                sample = seq_group.get('sample')
```

Comment:
very valid observation! since every sequencing group needs to have a sample, I'd actually be more inclined to error out here instead.  Happy to update this implementation to catch that error :)